orca-api/src/main/java/com/netflix/spinnaker/orca/pipeline/model/Execution.java

```diff
@@ -352,142 +351,4 @@ public static Execution newOrchestration(String application) {
   public static Execution newPipeline(String application) {
     return new Execution(PIPELINE, application);
   }
-
```

Comment:
What was the rationale behind making these root-level classes?

Reply:
To make things a little easier to deal with. I'm not that big of a fan of having static inner classes. This allows for things to be split a little bit better going forward. If we don't need some of those classes in the `orca-api`, we can put them back in `orca-core`

Reply:
@robzienert ^^

Reply:
I'm pretty gun-shy about inner classes, too. In this case however, the reason they are inner classes is because they are types that define a group of related attributes of a type, in this case `Execution`. They're not intended to be reused--and wouldn't necessarily fit--outside of `Execution`, which is why they're not root-level.

Reply:
This one seems problematic to me as it's significantly and unnecessarily blown out the size of this PR.

I'd like this style of change to be done independently.

Reply:
@robzienert @ajordens moved everything back into `Execution`. Can we get a marge please?